Source/WTF/wtf/glib/RunLoopGLib.cpp

```diff
@@ -86,48 +84,65 @@ RunLoop::RunLoop()
 RunLoop::~RunLoop()
 {
     g_source_destroy(m_source.get());
+    m_shouldStop = true;
+}
+
+void RunLoop::runGLibMainLoop(GMainContext* mainContext)
+{
+    g_main_context_ref(mainContext);
```

Comment:
You can use GRefPtr here.

Reply:
m_mainContext is already a GRefPtr, I think the manual ref/derefing is superfluous. Will double check, too.